src/util/ellipse/getEllipse.js

```diff
@@ -0,0 +1,19 @@
+import { getRadius } from '.';
+
+/**
+ * Retrieve the bounds of the ellipse in image coordinates
+ *
+ * @param {*} startHandle
+ * @param {*} endHandle
+ * @returns {{ left: number, top: number, width: number, height: number }}
+ */
+export default function _getCircleImageCoordinates(startHandle, endHandle) {
```

Comment:
You are calling this file `getEllipse` but it only has one radius, which makes it a circle, not an ellipse. Please change the name.

Reply:
Fixed!